src/_pytest/main.py

```diff
@@ -383,7 +391,7 @@ class Session(nodes.FSCollector):
     # Set on the session by fixtures.pytest_sessionstart.
     _fixturemanager = None  # type: FixtureManager
 
-    def __init__(self, config):
+    def __init__(self, config) -> None:
```

Comment:
Out of curiosity, is considered good practice to annotate the return value of all functions even if they return `None`?

Reply:
It has to be for the function to be considered to be typed.
I've amended it to also type `config` (the arg) explicitly, although done via base class.

Reply:
I see, thanks makes sense.

Now it brings the next question, is:

```python
def __init__(self, config: Config):
```

enough for it to be considered typed?

Reply:
Yes (AFAIK), but might fail with strict options etc?!  /cc @bluetech

Reply:
btw: `reveal_type(var)` is very useful in that regard.